### DISCUSSION TITLE
React 17 support for stable release

### DISCUSSION THREAD

**Discussion by maiertech** [system-ui/theme-ui#1352] [Q&A] [answered]
Current stable release v0.3.4 has this peer dependency:

```
"peerDependencies": {
    "react": "^16.11.0"
},
```

Is there a reason, why v17 has not been added yet?

I saw that React v17 has been added in later versions, so I am wondering if there is a later version with v17 support that is considered stable enough to be used a drop in replacement for v0.3.4?

**Comment by atanasster** [comment, marked as answer]
I am using heavily `"theme-ui": "^0.6.0-alpha.3"` on multiple projects and it works fine. I would recommend to use this and help us/you (we are also just volunteering community users like you) fix any deficiencies as we are heading to a 1.x version :)

As to why it has not been added to `0.3.4` stable -- just because the build/release process is a bit heavy and time-consuming.

**Reply by maiertech** [reply]
Will try.
